public/src/main/java/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -92,7 +92,12 @@ data class StoreTransaction(
      * In Google, this will be calculated from the basePlanId and offerId
      * Null for restored transactions and purchases initiated outside of the app.
      */
-    val subscriptionOptionId: String?
+    val subscriptionOptionId: String?,
+
+    /**
+     * The proration_mode string to be sent to the backend.
+     */
+    val prorationMode: String?
```

Comment:
Considering we are exposing the `ProrationMode` and `GoogleProrationMode` classes as well, any reason we don't use them here?

Reply:
Right... initially I didn't because of where I had put the `ProrationMode` interface. But I'll just change it.